CONTRIBUTING.md

```diff
@@ -89,150 +89,176 @@ effect on highlighting. We will work on improving highlighting in the near futur
 
 #### Misc
 
-```
-@comment
-@debug
-@error for error `ERROR` nodes.
-@none to disable completely the highlight
-@preproc
-@punctuation.delimiter for `;` `.` `,`
-@punctuation.bracket for `()` or `{}`
-@punctuation.special for symbols with special meaning like `{}` in string interpolation.
+```scheme
+@comment  ; line and block comments
+@debug    ; debugging statements [NOTE: unused]
```

Comment:
let's remove this, when nobody can give an example what is meant by debugging statements

Reply:
@theHamsta is the capture czar, but if it's not used by any current query, I'd suggest leaving it out for now; if somebody ends up needing it, they can easily add it (and provide a good description).